config/config.go

```diff
@@ -1102,6 +1102,7 @@ func setBidderDefaults(v *viper.Viper, bidder string) {
 	v.BindEnv(adapterCfgPrefix + ".usersync.redirect.redirect_url")
 	v.BindEnv(adapterCfgPrefix + ".usersync.redirect.external_url")
 	v.BindEnv(adapterCfgPrefix + ".usersync.redirect.user_macro")
+	v.BindEnv(adapterCfgPrefix + ".usersync.external_url")
```

Comment:
Can we add an assertion in `config/config_test.go`? Maybe in `TestFullConfig` or even in the `TestUnmarshalAdapterExtraInfo` test?

Reply:
Do you really think we need to? There's nothing special with this config value. Adding it to `TestFullConfig` is really just testing Viper and not so much this specific value.

Reply:
Sure, I can live with that.